frontend/app/.server/routes/helpers/renew-route-helpers.ts

```diff
@@ -101,6 +101,16 @@ export interface RenewState {
     acknowledgePrivacy: boolean;
     shareData: boolean;
   };
+  readonly isSurveyCompleted?: boolean;
+  readonly demographicSurvey?: {
```

Comment:
Do we need `isSurveyCompleted` if it can be determined based on whether `demographicSurvey` is defined or not?